packages.d/_ALWAYS.customise.add/etc/systemd/network/85-wlan-mesh.link

```diff
@@ -2,7 +2,7 @@
 # By default, all wlan interfaces will be access points
 [Match]
 Type=wlan
-Driver=rt2800usb
+Driver=ath9k_htc rt2800usb
```

Comment:
This change makes the below comment unclear.

Perhaps this change is a trigger to move this file out of the generic builder?